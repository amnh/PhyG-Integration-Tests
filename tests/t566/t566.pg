--input t89.tre, swap(inorder), inputting amino acid file
set(seed:1634561640)
read(prefasta:"metazoa-aa-1_missing.fas", tcm:"dayhoff.mat")
read(prefasta:"metazoa-aa-2_missing.fas", tcm:"dayhoff.mat")
read(newick:"../t89/t89.tre")
swap(inorder)
report("t566.csv", data, overwrite)
report("t566.tre", newick, graphs, overwrite)