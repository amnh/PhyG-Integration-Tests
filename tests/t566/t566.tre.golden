(Antarctic_neomeniomorph:0.0,(Greenland_neomeniomorph:47.0,((Helobdella_robusta:13.0,Scaphopods:5.0)HTU29:45.0,((Siphonaria_pectinata:2.0,(Octopus:48.0,(Lymnaea_stagnalis:0.0,(Cerebratulus_lacteus:56.0,Euprymna_scolopes:39.0)HTU41:0.0)HTU45:7.0)HTU28:0.0)HTU30:5.0,((Venerupis_decussatus:53.0,Venerupis_philippinarum:0.0)HTU27:3.0,(Lottia_gigantea:37.0,(((Crassostrea_virginica:6.0,Mytilus_californianus:2.0)HTU31:2.0,(Yoldia_limatula:0.0,(Nucula:15.0,Solemya:16.0)HTU38:0.0)HTU34:2.0)HTU36:0.0,((Ilyanassa_obsoleta:0.0,Nautilus:0.0)HTU40:0.0,(Perotrochus_lucaya:7.0,(Chaetopleura_apiculata:4.0,(Lingula_anatina:8.0,(Aplysia_californica:46.0,Littorina_littorea:14.0)HTU37:0.0)HTU35:15.0)HTU44:1.0)HTU43:9.0)HTU39:30.0)HTU25:2.0)HTU32:1.0)HTU33:2.0)HTU42:5.0)HTU46:4.0)HTU26:0.0)HTU24[1316.0];
(Antarctic_neomeniomorph:0.0,(Greenland_neomeniomorph:47.0,((Helobdella_robusta:13.0,Scaphopods:5.0)HTU29:45.0,((Siphonaria_pectinata:2.0,(Octopus:48.0,(Lymnaea_stagnalis:0.0,(Cerebratulus_lacteus:56.0,Euprymna_scolopes:39.0)HTU41:0.0)HTU45:7.0)HTU28:0.0)HTU30:5.0,((Venerupis_decussatus:53.0,Venerupis_philippinarum:0.0)HTU27:3.0,(Lottia_gigantea:37.0,((Yoldia_limatula:3.0,(Mytilus_californianus:32.0,(Crassostrea_virginica:0.0,(Nucula:15.0,Solemya:16.0)HTU38:0.0)HTU34:1.0)HTU31:2.0)HTU36:0.0,((Ilyanassa_obsoleta:0.0,Nautilus:0.0)HTU40:0.0,(Perotrochus_lucaya:7.0,(Chaetopleura_apiculata:4.0,(Lingula_anatina:8.0,(Aplysia_californica:46.0,Littorina_littorea:14.0)HTU37:0.0)HTU35:15.0)HTU44:1.0)HTU43:9.0)HTU39:30.0)HTU25:2.0)HTU32:1.0)HTU33:2.0)HTU42:5.0)HTU46:4.0)HTU26:0.0)HTU24[1316.0];
